tx/transaction.go

```diff
@@ -45,24 +46,24 @@ const (
 
 // Transaction is an immutable tx type.
 type Transaction struct {
-	body TxData
+	body txData
 
 	cache struct {
 		signingHash  atomic.Value
 		origin       atomic.Value
 		id           atomic.Value
 		unprovedWork atomic.Value
-		size         atomic.Value
-		intrinsicGas atomic.Value
+		size         atomic.Int64 // TODO: change to atomic.Uint64
```

Comment:
What needs to happen to switch it to `uint64`? Can you add that to the comment?

Reply:
Just a reminder for myself, planning to update them in another PR also looks into all possible atomic changes.